language-server/modules/langserver-core/src/test/resources/extensions/document/ast/modify/serviceNats.bal

```diff
@@ -1,18 +1,18 @@
-import ballerina/http;
-import ballerina/nats;
-@http:ServiceConfig {
-    basePath: "/"
-}
-service hello on new http:Listener(9090) {
-    @http:ResourceConfig {
-        methods: ["GET"],
-        path: "/sayHello"
-    }
-    resource function sayHello(http:Caller caller, http:Request req) {
-        nats:Connection connection = new ();
-        nats:Producer producer = new (connection);
-        nats:Error? result = producer->publish("Foo", "Test Message");
-
-    }
-}
+//import ballerina/http;
+//import ballerina/nats;
+//@http:ServiceConfig {
+//    basePath: "/"
+//}
+//service hello on new http:Listener(9090) {
+//    @http:ResourceConfig {
+//        methods: ["GET"],
+//        path: "/sayHello"
+//    }
+//    resource function sayHello(http:Caller caller, http:Request req) {
+//        nats:Connection connection = new ();
+//        nats:Producer producer = new (connection);
+//        nats:Error? result = producer->publish("Foo", "Test Message");
+//
+//    }
+//}
```

Comment:
Any reason to comment out these NATS related things rather removing?

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/25183

Reply:
I've created an issue to track it. @nadeeshaan will look into it.

Reply:
Ok, thanks.